test/mocks/MockERC1155.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {IERC1155} from "src/interfaces/IERC1155.sol";
+
+contract MockERC1155 is IERC1155 {
```

Comment:
Why not inherit OZ's implementation to mock. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/ERC1155.sol

Reply:
The reason was we just needed to setBalance , figured it would make the mock a little messy and might be easier to just have those functions we use in the mod. You reckon it's better to inherit the OZ contract and implement ?

Reply:
Yes, I don't think the integration would be too bad and the tests would be a little safer. The contract would now be a fake instead of a mock. We typically use this pattern when testing with Erc20s.